### ISSUE TITLE
Plugin-scanning question dialog is not accessible

### ISSUE THREAD

**Issue by saintmatthieu** [audacity/audacity#9353]
### Bug description

When starting the app and there are some plugins that need scan, now we have this dialog asking whether scanning should happen now or not.
Problem: it is not accessible (Escape, tab, ..., nothing works)

### Steps to reproduce

1. install a new plugin or delete known_audio_plugins.json
2. start audacity and observe the plugin-scan dialog

### Expected behavior

Dialog can be controlled with the keyboard.

### Actual behavior

It's not.

### Audacity Version

master

### Operating system

OS independent

### Additional context

Devs can look at this message to get started (restricted access): https://mu--se.slack.com/archives/C06RX437GJV/p1757499413802479?thread_ts=1757494092.121159&cid=C06RX437GJV

**Comment by chinakov** [comment]
Still present on the latest Master (47ecf3eeabc46945542e87107355900f1fd5c326)

**Comment by dozzzzer** [comment]
Tested
